src/libs/NextStepUtils.ts

```diff
@@ -121,7 +116,14 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
             if (harvesting?.enabled && autoReportingFrequency !== CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL) {
                 optimisticNextStep.message = [
                     {
-                        text: 'These expenses are scheduled to ',
+                        text: 'Waiting for ',
+                    },
+                    {
+                        text: `${ownerDisplayName}`,
```

Comment:
```suggestion
                        text: `${ownerDisplayName}'s `,
```

Otherwise the "'s" part will not be **bold**

Reply:
@eVoloshchak @rayane-djouah the `'s` part will not be bold